distributed/http/scheduler/tests/test_semaphore_http.py

```diff
@@ -17,6 +17,9 @@ async def test_prometheus_collect_task_states(c, s, a, b):
     async def fetch_metrics():
         port = s.http_server.port
         response = await http_client.fetch(f"http://localhost:{port}/metrics")
+        assert response.code == 200
+        assert response.headers["Content-Type"] == "text/plain; version=0.0.4"
```

Comment:
Do we need to assert the version? I'd think it's okay if that changes?

Reply:
Dropped during refactoring.